Makefile

```diff
@@ -199,6 +201,12 @@ checkpatch:
 			| ${CHECKPATCH} - || true;			\
 	done
 
+# Target used to check for suspiciously missing changed files.
+
+checkdiff:
+	$Qeval COMMON_COMMIT=$$(git merge-base HEAD ${BASE_REF});	\
```

Comment:
Come to think of it, what is that `eval` for?

Reply:
Copied from the `checkpatch` rule; is it not really necessary to assign `COMMON_COMMIT`?

Reply:
I don't think so... besides, `$()` syntax is Bash, not POSIX `sh`, so we probably should change this as well.

Reply:
Assigning is required for `checkpatch` since it's used within backticks, but it shouldn't be necessary here.